lib/AST/ASTContext.cpp

```diff
@@ -1916,6 +1916,29 @@ ASTContext::getModule(ImportPath::Module ModulePath) {
   return nullptr;
 }
 
+ModuleDecl *ASTContext::getOverlayModule(const FileUnit *FU) {
+  assert(FU && FU->getKind() == FileUnitKind::ClangModule &&
+         "Overlays can only be retrieved for clang modules!");
+
+  ImportPath::Module::Builder builder(FU->getParentModule()->getName());
+  auto ModPath = builder.get();
+  if (auto *Existing = getLoadedModule(ModPath)) {
+    if (!Existing->isNonSwiftModule())
```

Comment:
Any chance of adding an `isSwiftModule() { return !isNotSwiftModule() }` ? That way a reader would not have to mentally decode the double negative.